addons/web_editor/static/src/scss/wysiwyg_iframe.scss

```diff
@@ -30,7 +31,6 @@ body.o_in_iframe {
 
     .iframe-editor-wrapper {
         flex: 1 1 auto;
-        overflow: auto;
```

Comment:
Yes. Do not change overflow values in mass mailing as it may break the scrolling in one of its many contexts. I think this particular one might break it in a modal.

Reply:
I've modified the commit to remove the overflow in .note-editable only when the dropzone are on the page.

Reply:
I don't get it.

```css
&.oe_zone_active .note-editable {
        overflow: hidden;
}
```

1. `.note-editable` is not a child of the dropzones so this would never do anything
2. I looked for `.oe_zone_active` in the dom when the dropzones were active and couldn't find anything.
3. I grepped `oe_zone_active` in the codebase and couldn't find anything.